server/memo.go

```diff
@@ -702,3 +710,49 @@ func (s *Server) getMemoDisplayWithUpdatedTsSettingValue(ctx context.Context) (b
 	}
 	return memoDisplayWithUpdatedTs, nil
 }
+
+// FilterPublicMemosByDays filter public MemoMessages by creator's public-in-days setting.
+func FilterPublicMemosByDays(ctx context.Context, s *store.Store, list []*store.MemoMessage) ([]*store.MemoMessage, error) {
```

Comment:
Do we need to call this function in other package? If not let's naming it in lower case `filterPublicMemosByDays`.